README.md

```diff
@@ -178,18 +178,18 @@ We informally say that a representation generalizes well from a training to a te
 # Installation
 - Create and activate a conda virtual environment:
   ```
-  conda create -n segar pip python=3.6
+  conda create -n segar pip python=3.9
   conda activate segar
   ```
 - Clone the current repo and install it in editable mode:
   ```
-  git clone https://github.com/microsoft/roboputtputt.git
-  cd roboputtputt
+  git clone https://github.com/microsoft/segar.git
+  cd segar
   pip install -e .
   ```
 - For running RL samples (using rllib):
   ```
-  pip install -e '.[rl]'
+  pip install -e .[rl]
```

Comment:
no, this should stay in quotation marks because otherwise on many non-BASH shells this doesn't work. `[` and `]` are special characters on most shells (see e.g. here: https://unix.stackexchange.com/questions/296141/how-to-use-a-special-character-as-a-normal-one-in-unix-shells)

Reply:
ok.